library/L2_HAL/actuators/servo/test/servo_test.cpp

```diff
@@ -0,0 +1,78 @@
+#include "L0_Platform/lpc40xx/LPC40xx.h"
+#include "L2_HAL/actuators/servo/servo.hpp"
+#include "L4_Testing/testing_frameworks.hpp"
+
+namespace sjsu
+{
+EMIT_ALL_METHODS(Servo);
+
+TEST_CASE("Testing Servo", "[Servo]")
+{
+  // Make a mock pin to work with
+  Mock<Pwm> mock_pwm;
+
+  // Fake the implementation of:
+  // Initialize
+  // SetFrequency
+  // SetDutyCycle

```

Comment:
You can remove the above lines. The test code is quite expressive enough for a reader to understand what is going on.